app/build.gradle

```
@@ -30,6 +34,9 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    dataBinding{
+        enabled = true
+    }
```

```suggestion
    buildFeatures{
         dataBinding = true
    }
```
Android Studio 4.0 이상 부터는 Android Gradle Plugin가 아닌 buildFeatures로 param이 이동되었기 때문에 수정하는게 좋을듯 합니다!